edx_exams/apps/api/v1/tests/test_views.py

```diff
@@ -256,3 +256,34 @@ def test_exams_config(self, other_course_id, exam_type, expect_none_provider):
         self.assertEqual(new_exam.due_date, pytz.utc.localize(datetime.fromisoformat('2025-07-01 00:00:00')))
         self.assertEqual(new_exam.hide_after_due, False)
         self.assertEqual(new_exam.is_active, True)
+
+
+@ddt.ddt
```

Comment:
I don't think we need the data driven testing decorator on the class if the tests don't actually use any ddt decorators